java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java

```diff
@@ -0,0 +1,82 @@
+package org.openqa.selenium.grid.distributor.local;
```

Comment:
Please add the project copyright at the top. `./go copyright:update` can do that for you.